multipath/multipath.conf

```diff
@@ -1,3 +1,11 @@
+# --- WARNING: DO NOT EDIT THIS FILE ---
+# The contents of this file may be overwritten at any future time through a
+# system update, causing any custom configuration to be lost.
+#
+# For custom multipath configuration, create a separate .conf file in the
+# /etc/multipath/conf.d/ directory.
```

Comment:
To be changed into an invitation to modify the /etc/multipath/conf.d/custom.conf if my proposal consisting in installing such a file by default on the system is accepted.

Reply:
I think it's better for vendor neutrality for the configuration overrides to go into separate files.

Reply:
I'm not sure about what you mean. Since the PR also provides a ready-to-modify file to be placed in `/etc/multipath/conf.d/custom.conf `, wouldn't it be appropriate to mention it?